compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirMethodOfAnyImplementedInInterfaceChecker.kt

```diff
@@ -61,9 +48,11 @@ object FirMethodOfAnyImplementedInInterfaceChecker : FirRegularClassChecker(), F
         }
 
         for (it in declaration.declarations) {
+            if (it !is FirSimpleFunction || !it.isOverride || !it.hasBody) continue
```

Comment:
Hm, I don't think it will make any differences. That part is one-time operation to collect all public APIs in `Any`. And then, here, we check if any function override in interfaces conflict with `Any`'s member. I don't think nothing will be filtered out with override/body check for members of `Any`.